contracts/deployables/modules/AzoriusV1.sol

```diff
@@ -488,43 +341,21 @@ contract AzoriusV1 is IAzoriusV1, GuardableModule, Version, UUPSUpgradeable {
         if (!exec(_target, _value, _data, _operation)) revert TxFailed();
     }
 
-    /**
-     * Enables the specified array of [BaseStrategy](./BaseStrategy.md) contract addresses.
-     *
-     * @param _strategies array of `BaseStrategy` contract addresses to enable
-     */
-    function _setUpStrategies(address[] memory _strategies) internal {
-        strategies[SENTINEL_STRATEGY] = SENTINEL_STRATEGY;
-        uint256 strategiesLength = _strategies.length;
-        for (uint256 i; i < strategiesLength; ) {
-            enableStrategy(_strategies[i]);
-            unchecked {
-                ++i;
-            }
-        }
-    }
-
-    /**
-     * Updates the `timelockPeriod` for future Proposals.
-     *
-     * @param _timelockPeriod new timelock period
-     */
     function _updateTimelockPeriod(uint32 _timelockPeriod) internal {
         timelockPeriod = _timelockPeriod;
         emit TimelockPeriodUpdated(_timelockPeriod);
     }
 
-    /**
-     * Updates the `executionPeriod` for future Proposals.
-     *
-     * @param _executionPeriod new execution period
-     */
     function _updateExecutionPeriod(uint32 _executionPeriod) internal {
         executionPeriod = _executionPeriod;
         emit ExecutionPeriodUpdated(_executionPeriod);
     }
 
-    /// Implementation for the version
+    function _updateStrategy(address _strategy) internal {
+        strategy = IStrategyBaseV1(_strategy);
```

Comment:
add zero address check for strategy? like we used to check for `enableStrategy` as below code:
```solidity
function enableStrategy(address _strategy) public override onlyOwner {
        if (_strategy == address(0) || _strategy == SENTINEL_STRATEGY)
            revert InvalidStrategy();
```

Reply:
Good call, will do.